preview/Win7Msix/Win7MSIXInstaller/MsixResponse.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include <windows.h>
+#include <string>
+#include "GeneralUtil.hpp"
+
+class MsixResponse
+{
+private:
+
+    ///Variable used to indicate if add package request was cancelled during installation
+    bool m_isInstallCancelled = false;
+
+    HRESULT errorCode;
+
+    PCWSTR errorText;
+
+public:
+
+    static HRESULT Make(MsixResponse** outInstance);
+
+    //Setters and Getters

```

Comment:
>//Setters and Getters [](start = 4, length = 21)

Follow standard comments like we discussed with /// and more information about parameters, return values, potential errors etc.

Reply:
Added comments.